src/main/java/com/spotinst/sdkjava/model/ApiBlockDevice.java

```diff
@@ -15,7 +15,7 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @JsonFilter("PartialUpdateEntityFilter")
-class ApiBlockDevice implements IPartialUpdateEntity {
+public class ApiBlockDevice implements IPartialUpdateEntity {
```

Comment:
Why this file is used from the sdkjava/model. Should we create a file specific to ocean AKS?